### DISCUSSION TITLE
Mobile users are bouncing!

### DISCUSSION THREAD

**Discussion by ben519** [squidfunk/mkdocs-material#3721] [General]
When mobile users visit my site, they frequently navigate to a single page and then bounce! Has anyone else noticed this issue?

Most of my rich content exists in three or four levels down in the navigation tree

```
- Home
- Boring Section 1
  - Rich Article 1
  - Rich Article 2
- Boring Section 2
  - Rich Article 3
  - Rich article 4 
```

My mobile users tend to reach Boring Section 1 and/or Boring Section 2, never making it to Rich Article 1 or Rich Article 2 which is *really weird* because that's where the juicy content lives, and presumably that's what they came to my site intending to read.

I asked a few users about this who said they didn't realize that they could navigate multiple levels deep.

Curious to hear if other people have experienced this and how you've dealt with this issue. I assume the easy solution is just to put links to my deeper pages inside the content of the higher pages, but this seems a bit redundant given the auto-generating navigation bar.

**Comment by squidfunk** [comment]
Interesting. The navigation is hidden behind the hamburger icon. This should be expected by users after years of apps and app-like websites. However, you might need to add links to guide users, if that's not enough. Really depends on the use case though.

**Reply by ben519** [reply]
Thanks. That's what I ended up doing and people are bouncing much less.
